Formula/grantlee.rb

```diff
@@ -0,0 +1,44 @@
+class Grantlee < Formula
+  desc "Libraries for text templating with Qt"
+  homepage "https://github.com/steveire/grantlee"
+  url "https://github.com/steveire/grantlee/releases/download/v5.3.1/grantlee-5.3.1.tar.gz"
+  sha256 "ba288ae9ed37ec0c3622ceb40ae1f7e1e6b2ea89216ad8587f0863d64be24f06"
+  license "LGPL-2.1-or-later"
+  head "https://github.com/steveire/grantlee.git", branch: "master"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "doxygen" => :build
+  depends_on "graphviz" => :build
+
+  depends_on "qt@5"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```

Comment:
in case we can build/test with qt6 later
```suggestion
    args = %w[
      -DBUILD_TESTS=OFF
      -DGRANTLEE_BUILD_WITH_QT6=ON
    ]

    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
```